components/changjiang-electronics-tech/SS8050.stanza

```diff
@@ -13,7 +13,6 @@ defpackage ocdb/components/changjiang-electronics-tech/SS8050 :
   import ocdb/utils/symbols
 
   import ocdb/utils/property-structs
-  import ocdb/utils/generator-utils
```

Comment:
This was a duplicate import (from a few lines up).

It started failing CI after updating to latest jitx-client docker image - maybe new logic to prevent these?